test/validations.test.js

```diff
@@ -322,6 +322,43 @@ describe('validations', function() {
       });
     });
 
+    it('should pass options to custom synch', function(done) {
+      delete User.validations;
+      User.validate('name', function(err, options) {
+        if (options.options !== 'someValue') err();
+      });
+      User.create({name: 'Valid'}, {options: 'someValue'}, function(e, d) {
+        d.updateAttribute('name', null, {options: 'otherValue'}, function(e) {
+          should.exist(e);
+          e.should.be.instanceOf(Error);
```

Comment:
I think this check is redundant considering that the next check is `instanceOf(ValidationError)`.